File: lambda-api/MyService/MXU.ts
import {BaseMessage as $UniRpc_BaseMessage} from "../UniRpc/BaseMessage";
import {ReturnMessage as $UniRpc_ReturnMessage} from "../UniRpc/ReturnMessage";
import {WebsocketServiceBase as $UniRpc_WebsocketServiceBase} from "../UniRpc/WebsocketServiceBase";
import {IMXService as $MyService_IMXService} from "./IMXService";
import {IUXService as $MyService_IUXService} from "./IUXService";
export abstract class MXU extends $UniRpc_WebsocketServiceBase implements $MyService_IMXService, $MyService_IUXService
{
    public constructor () {
        super();
        this.__reflection = "MyService.MXU";
    }
    public abstract testJob(): Promise<boolean>;
    /**
     * comment 2 is not
     * just one line
     */
    public abstract resolve(): Promise<void>;
    public abstract test(): Promise<Promise<string>>;
    public async __invoke(message: $UniRpc_BaseMessage): Promise<$UniRpc_BaseMessage> {
        switch (message.Method)
        {
            case "testJob":
                {
                    return $UniRpc_ReturnMessage(message, await this.testJob());
                }
            case "resolve":
                {
                    await this.resolve();
                    break;
                }
            case "test":
                {
                    return $UniRpc_ReturnMessage(message, await this.test());
                }
        }
        throw `${message.Service}.${message.Method} is not defined.`;
    }
}

File: pywebsocket/MyService/MXU.py
from abc import ABC, abstractmethod
from MyService.IMXService import IMXService as ImportedRef__MyService_IMXService
from MyService.IUXService import IUXService as ImportedRef__MyService_IUXService
from UniRpc.BaseMessage import BaseMessage as ImportedRef__UniRpc_BaseMessage
from UniRpc.ReturnMessage import ReturnMessage as ImportedRef__UniRpc_ReturnMessage
from UniRpc.WebsocketServiceBase import WebsocketServiceBase as ImportedRef__UniRpc_WebsocketServiceBase

class MXU(ImportedRef__UniRpc_WebsocketServiceBase, ImportedRef__MyService_IMXService, ImportedRef__MyService_IUXService):
    def __init__(self):
        super().__init__()
        self.__reflection = 'MyService.MXU'

    @abstractmethod
    async def testJob(self) -> bool:
        pass

    @abstractmethod
    async def resolve(self) -> None:
        '''
        comment 2 is not
        just one line
        '''
        pass

    @abstractmethod
    async def test(self) -> str:
        pass

    async def __invoke(self, message: ImportedRef__UniRpc_BaseMessage) -> ImportedRef__UniRpc_BaseMessage:
        if message.Method == 'testJob':
            return ImportedRef__UniRpc_ReturnMessage(message, await self.testJob())
        elif message.Method == 'resolve':
            await self.resolve()
        elif message.Method == 'test':
            return ImportedRef__UniRpc_ReturnMessage(message, await self.test())
        else:
            raise Exception(f'{message.Service}.{message.Method} is not defined.')